app/javascript/tests/public_health/actions/UpdateCaseStatus.test.js

```diff
@@ -0,0 +1,210 @@
+    expect(mountedWrapper.find('.modal-body').find('p').at(0).text()).toContain('Please select the desired case status to be assigned to all selected patients:')
+    mountedWrapper.find('#case_status').find('option').forEach((option, index) => {
+      expect(option.text()).toEqual(CASE_STATUS_OPTIONS[index]);
+    });
+    expect(mountedWrapper.find('.modal-body').find('p').at(1).text()).toContain('The selected cases will remain in the exposure workflow.')
+    expect(mountedWrapper.find('ModalFooter').find('Button').at(0).text()).toEqual('Cancel')
+    expect(mountedWrapper.find('ModalFooter').find('Button').at(1).text()).toEqual('Submit')
+  });
+
+  it('Properly renders all main components for Patients who are in Isolation only', () => {
+    // mockPatien1 and mockPatient4 are both in the Isolation workflow
+    const mountedWrapper = getMountedWrapper([mockPatient1, mockPatient4]);
+    expect(mountedWrapper.find('.modal-body').find('p').at(0).text()).toContain('Please select the desired case status to be assigned to all selected patients:')
+    mountedWrapper.find('#case_status').find('option').forEach((option, index) => {
+      expect(option.text()).toEqual(CASE_STATUS_OPTIONS[index]);
+    });
+    expect(mountedWrapper.find('.modal-body').find('p').at(1).text()).toContain('The selected cases will be moved from the isolation workflow to the exposure workflow and placed in the symptomatic, non-reporting, or asymptomatic line list as appropriate.')
+    expect(mountedWrapper.find('ModalFooter').find('Button').at(0).text()).toEqual('Cancel')
+    expect(mountedWrapper.find('ModalFooter').find('Button').at(1).text()).toEqual('Submit')
+  });
+
+  it('Properly renders all main components for Patients who are in both workflows', () => {
+    const mountedWrapper = getMountedWrapper([mockPatient1, mockPatient2, mockPatient3, mockPatient4]);
+    expect(mountedWrapper.find('.modal-body').find('p').at(0).text()).toContain('Please select the desired case status to be assigned to all selected patients:')
+    mountedWrapper.find('#case_status').find('option').forEach((option, index) => {
+      expect(option.text()).toEqual(CASE_STATUS_OPTIONS[index]);
+    });
+    // The behavior here is the same as if they were in the isolation workflow
+    expect(mountedWrapper.find('.modal-body').find('p').at(1).text()).toContain('The selected cases will be moved from the isolation workflow to the exposure workflow and placed in the symptomatic, non-reporting, or asymptomatic line list as appropriate.')
+    expect(mountedWrapper.find('ModalFooter').find('Button').at(0).text()).toEqual('Cancel')
+    expect(mountedWrapper.find('ModalFooter').find('Button').at(1).text()).toEqual('Submit')
+  });
+
+  it('Properly does not display any follow-up options when the empty string is selected', () => {
+    // have to `mount` the wrapper so the event object is properly created when simulating change below
```

Comment:
you should be able to do this with shallow, lmk if you want some help looking at this